eras/alonzo/impl/testlib/Test/Cardano/Ledger/Alonzo/Imp/UtxosSpec.hs

```diff
@@ -69,3 +81,22 @@ spec = describe "UTXOS" $
                           (Data $ P.I 32, exUnits)
                       )
               )
+        describe "Scripts pass in phase 2" $ do
+          let scripts' = drop 1 scripts
+          forM_ scripts' $ \(name, script) -> do
+            it name $ do
+              let sHash = hashPlutusScript (script slang)
+              txIn0 <- produceScript sHash
+              exUnits <- getsNES $ nesEsL . curPParamsEpochStateL . ppMaxTxExUnitsL
```

Comment:
Using max execution unit is incorrect because that would prevent having more than one script in the tx.

Also it is in unnecessary, since fixup functionality will take care of it